sklearn/naive_bayes.py

```diff
@@ -577,30 +657,6 @@ def _check_X_y(self, X, y, reset=True):
         """Validate X and y in fit methods."""
         return self._validate_data(X, y, accept_sparse="csr", reset=reset)
 
-    def _update_class_log_prior(self, class_prior=None):
```

Comment:
This function moved in `_BaseNB` since it could be used by all naive Bayes algorithm